microcloud/cmd/microcloud/main_init_preseed.go

```diff
@@ -391,6 +392,22 @@ func (p *Preseed) Parse(s *service.Handler, bootstrap bool) (map[string]InitSyst
 		return nil, err
 	}
 
+	expectedServices := make(map[types.ServiceType]service.Service, len(s.Services))
+	for k, v := range s.Services {
+		expectedServices[k] = v
+	}
+
+	for serviceType := range expectedServices {
+		initializedSystem, _, err := checkClustered(s, false, serviceType, systems)
+		if err != nil {
+			return nil, err
+		}
+
+		if initializedSystem != "" && !p.ReuseExistingClusters {
+			delete(s.Services, serviceType)
+		}
```

Comment:
I'm not sure if we should error out here or not. I wouldn't expect ceph to be running on a LXD cluster member unless it was being used in the that LXD cluster, so for me this would be indicative of misconfiguration.

Reply:
It's tough since MicroCeph/MicroOVN might have other applications besides LXD. 

If we error out, that's like saying "you can't have a MicroCloud with these systems until you fundamentally change your current MicroCeph cluster". If we don't error out, it's "You can still have a MicroCloud, but it won't include MicroCeph until you fundamentally change your current MicroCeph cluster".

Since MicroCeph and MicroOVN are optional for MicroCloud, I think the second one makes sense and the first is too restrictive. PR #260 which builds on this PR will allow for adding MicroCeph/MicroOVN to MicroCloud if it was optionally/implicitly skipped before.

Reply:
I've also added a warning that the service will be skipped now, so the user is aware of it.